src/controllers/RideOffer.js

```diff
@@ -1,52 +1,109 @@
 import Model from '../models/model';
 
 const driverOfferModel = new Model('ride_offer');
+/**
+ *
+ * @param {object} req - request
+ *
+ * @param {object} res - response
+ *
+ * @return {obj} Added data to database back
+ */
 export const addOffer = async (req, res) => {
+  const { id } = req.user.data;
   const {
     driverId, destination, price
   } = req.body;
   const columns = 'driver_id, destination, price';
-  const values = `'${driverId}', '${destination}', '${price}'`;
+  const values = `'${driverId}','${destination}', '${price}'`;
   try {
     const data = await driverOfferModel.insertWithReturn(columns, values);
-    res.status(200).json({ msg: data.rows });
+    // if (id === data.rows.driver_id) {
```

Comment:
If this code is no longer needed, clear it out completely instead of commenting it out.